cypress/integration/visitor_can_see_menu.feature.js

```diff
@@ -0,0 +1,29 @@
+describe("User can see menu", () => {
+  beforeEach(() => {
+    cy.server();
+  });
+
+  it("visitor can see menu", () => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3001/api/v1/products",
+      response: "fixture:products.json",
+    });
+    cy.visit("http://localhost:3001/");
+    cy.get("button#add").should("be.visible");
+    cy.get("button#minus").should("be.visible");
+    cy.get("td#numberOfItems").should("be.visible");
+    cy.get("button#plus").should("be.visible");
```

Comment:
Technically, the buttons aren't within scope for this first feature I think, but I don't know if they should be removed, now that they're there.

Reply:
please remove them, out of scope

We are testing the functionality for seeing a menu here, not seeing add to order buttons